src/flamenco/gossip/fd_crds_value.h

```diff
@@ -2,5 +2,65 @@
 #define HEADER_fd_src_flamenco_gossip_fd_crds_value_h
 
 #include "../../util/fd_util.h"
+#include "../../util/net/fd_net_headers.h"
+
+#define FD_GOSSIP_CONTACT_INFO_SOCKET_GOSSIP            ( 0)
+#define FD_GOSSIP_CONTACT_INFO_SOCKET_SERVE_REPAIR_QUIC ( 1)
+#define FD_GOSSIP_CONTACT_INFO_SOCKET_RPC               ( 2)
+#define FD_GOSSIP_CONTACT_INFO_SOCKET_RPC_PUBSUB        ( 3)
+#define FD_GOSSIP_CONTACT_INFO_SOCKET_SERVE_REPAIR      ( 4)
+#define FD_GOSSIP_CONTACT_INFO_SOCKET_TPU               ( 5)
+#define FD_GOSSIP_CONTACT_INFO_SOCKET_TPU_FORWARDS      ( 6)
+#define FD_GOSSIP_CONTACT_INFO_SOCKET_TPU_FORWARDS_QUIC ( 7)
+#define FD_GOSSIP_CONTACT_INFO_SOCKET_TPU_QUIC          ( 8)
+#define FD_GOSSIP_CONTACT_INFO_SOCKET_TPU_VOTE          ( 9)
+#define FD_GOSSIP_CONTACT_INFO_SOCKET_TVU               (10)
+#define FD_GOSSIP_CONTACT_INFO_SOCKET_TVU_QUIC          (11)
+#define FD_GOSSIP_CONTACT_INFO_SOCKET_TPU_VOTE_QUIC     (12)
+
+#define FD_GOSSIP_CLIENT_SOLANA (0)
+#define FD_GOSSIP_CLIENT_JITO   (1)
+#define FD_GOSSIP_CLIENT_FD     (2)
+#define FD_GOSSIP_CLIENT_AGAVE  (3)
+
+struct fd_gossip_contact_info {
```

Comment:
I would namespace these,

```suggestion
struct fd_gossip_message_contact_info {
```